core/src/main/java/io/questdb/network/NetworkFacadeImpl.java

```diff
@@ -280,30 +279,6 @@ public boolean testConnection(int fd, long buffer, int bufferSize) {
         }
 
         final int nRead = Net.peek(fd, buffer, bufferSize);
-
-        if (nRead == 0) {
-            return false;
-        }
-
-        if (nRead < 0) {
-            return true;
-        }
-
-        // Read \r\n from the input stream and discard it since some HTTP clients
-        // send these chars as a keep alive in between requests.
-        int index = 0;
-        while (index < nRead) {
-            byte b = Unsafe.getUnsafe().getByte(buffer + index);
-            if (b != (byte) '\r' && b != (byte) '\n') {
-                break;
-            }
-            index++;
-        }
-
-        if (index > 0) {
-            Net.recv(fd, buffer, index);
```

Comment:
Note: I had to remove this HTTP-specific logic since it would break encrypted comms. @bluestreak01 please let me know if you think we should keep it. In this case, we'll need to introduce a separate impl that would accept a `Socket`.

Reply:
this is ok to removed from generic interface. Where did it go? :)

Reply:
Well, considering that no tests failed after I removed that code, it could be dead code, so that's why I'm double checking.